projects/js-packages/social-previews/composer.json

```
@@ -0,0 +1,48 @@
+{
+	"name": "automattic/social-previews",
+	"description": "A suite of components to generate previews for a post for both social and search engines.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"build-development": [
+			"pnpm run build"
+		],
+		"build-production": [
+			"NODE_ENV=production pnpm run build"
+		],
+		"test-coverage": [
+			"pnpm run test-coverage"
+		],
+		"test-js": [
+			"pnpm run test"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"autotagger": true,
+		"textdomain": "jetpack-social-previews",
```

If you want to keep it the same as before, the text domain could be 
```suggestion
		"textdomain": "social-previews",
```